app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -514,51 +503,45 @@ $after-button-padding-left: govuk-spacing(4);
       }
     }
 
-    // stylelint-enable max-nesting-depth
-  }
-}
-
-.gem-c-layout-super-navigation-header__navigation-top-toggle-button.gem-c-layout-super-navigation-header__navigation-top-toggle-button--blue-background,
-.gem-c-layout-super-navigation-header__navigation-top-toggle-button {
```

Comment:
It does make sense to remove this to me, but when testing the blue background variation of the component on Safari for iOS it will cause the "Menu" text for the button to appear white instead of blue when clicked, for example:

<img width="451" alt="iOS-menu-example" src="https://github.com/user-attachments/assets/3934e62a-d5e6-47d0-b949-522bd0453220">

Likely several ways to solve this, possibly re-ordering the CSS to help set the expected precedence to avoid adding further CSS. This could take a bit of work and likely out of scope for the original intended purpose of code cleanup in the PR.